models/order.js

```diff
@@ -27,6 +28,20 @@ orderSchema.statics.updateEntry = function(orderId, entryIndex, update) {
     });
 }
 
+orderSchema.statics.addOrder = function(entries) {
+  return this({
+    id: faker.random.number(), // need auto generated id...
+    entries: entries.map((item) => ({
+      item_id: mongoose.Types.ObjectId(item.id),
+      comment: item.comment
+    }))
+  })
+  .save()
+  .then((order) => {
```

Comment:
Nit: I think we could collapse to one line to make slightly cleaner without braces and return.

`.then((order) => populateEntries(toOrder(order));`